[2024-04-29T07:28:58.820+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-04-29T07:28:58.851+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_for_open_weather.weather_data scheduled__2024-04-29T03:16:00+00:00 [queued]>
[2024-04-29T07:28:58.865+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_for_open_weather.weather_data scheduled__2024-04-29T03:16:00+00:00 [queued]>
[2024-04-29T07:28:58.868+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 2
[2024-04-29T07:28:58.889+0000] {taskinstance.py:2327} INFO - Executing <Task(HttpOperator): weather_data> on 2024-04-29 03:16:00+00:00
[2024-04-29T07:28:58.902+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=1669) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-04-29T07:28:58.904+0000] {standard_task_runner.py:63} INFO - Started process 1703 to run task
[2024-04-29T07:28:58.905+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'etl_for_open_weather', 'weather_data', 'scheduled__2024-04-29T03:16:00+00:00', '--job-id', '403', '--raw', '--subdir', 'DAGS_FOLDER/open_weather.py', '--cfg-path', '/tmp/tmp9bm8wr7p']
[2024-04-29T07:28:58.906+0000] {standard_task_runner.py:91} INFO - Job 403: Subtask weather_data
[2024-04-29T07:28:58.975+0000] {task_command.py:426} INFO - Running <TaskInstance: etl_for_open_weather.weather_data scheduled__2024-04-29T03:16:00+00:00 [running]> on host 6dbc7f679521
[2024-04-29T07:28:59.101+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_for_open_weather' AIRFLOW_CTX_TASK_ID='weather_data' AIRFLOW_CTX_EXECUTION_DATE='2024-04-29T03:16:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-04-29T03:16:00+00:00'
[2024-04-29T07:28:59.103+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-04-29T07:28:59.103+0000] {http.py:169} INFO - Calling HTTP method
[2024-04-29T07:28:59.113+0000] {base.py:84} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-04-29T07:28:59.123+0000] {base.py:84} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-04-29T07:28:59.531+0000] {http.py:208} INFO - {"coord":{"lon":49.892,"lat":40.3777},"weather":[{"id":800,"main":"Clear","description":"clear sky","icon":"01d"}],"base":"stations","main":{"temp":298.18,"feels_like":297.68,"temp_min":298.18,"temp_max":298.18,"pressure":1020,"humidity":36},"visibility":10000,"wind":{"speed":9.26,"deg":150},"clouds":{"all":0},"dt":1714375528,"sys":{"type":1,"id":8841,"country":"AZ","sunrise":1714354919,"sunset":1714404798},"timezone":14400,"id":587084,"name":"Baku","cod":200}
[2024-04-29T07:28:59.534+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-04-29T07:28:59.562+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=etl_for_open_weather, task_id=weather_data, execution_date=20240429T031600, start_date=20240429T072858, end_date=20240429T072859
[2024-04-29T07:28:59.764+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-04-29T07:28:59.789+0000] {taskinstance.py:3482} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-04-29T07:28:59.793+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
